plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -112,7 +113,7 @@ public static ColumnMapping tinyintColumnMapping()
 
     public static LongWriteFunction tinyintWriteFunction()
     {
-        return (statement, index, value) -> statement.setByte(index, SignedBytes.checkedCast(value));
+        return LongWriteFunction.of(Types.TINYINT, (statement, index, value) -> statement.setByte(index, SignedBytes.checkedCast(value)));
```

Comment:
Can it happen that the remote database doesn't support e.g. Types.TINYINT?

Reply:
If it does - it will be caught by type mapping tests - it wasn't